src/components/BounceRuleDetailed/Modals/index.jsx

```diff
@@ -0,0 +1,188 @@
+import React from "react";
+import CenterModal from "@sendgrid/ui-components/center-modal";
+import Button from "@sendgrid/ui-components/button";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+} from "@sendgrid/ui-components/table/table";
+import { Column } from "../../Column";
+import { Row } from "../../Row";
+
+const ConfirmModalBody = ({
+  isConfirmOpen,
+  isCancelConfirmOpen,
+  handleModalClose,
+  handleModalConfirm,
+  modalType,
+}) => (
+  <div>
+    <Row>
+      <Column>
+        {(isConfirmOpen && (
+          <div>
+            <h2>Are you sure you&apos;d like to commit these changes?</h2>
+            <p>
+              {
+                "Doing so will effect how current email will be handled via this\
+                bounce rule. This action will go into effect immediately."
+              }
+            </p>
+          </div>
+        )) ||
```

Comment:
let's remove the `||` since it's not necessary.  {isConfirmOpen && (...)} and {isCancelConfirmOpen && (...)} are already doing the checking

Reply:
oops this is outdated. you can ignore this